modules/management-console-agent/src/test/java/org/apache/ignite/agent/processor/CacheChangesProcessorTest.java

```diff
@@ -61,6 +63,45 @@ public void shouldSendInitialStates() throws Exception {
         assertWithPoll(() -> interceptor.getPayload(buildClusterCachesSqlMetaDest(cluster.id())) != null);
     }
 
+    /** */
+    @Test
+    public void shouldSendValidCacheInfo() throws Exception {
+        IgniteEx ignite = (IgniteEx) startGrid();
+
+        changeManagementConsoleConfig(ignite);
+
+        IgniteCluster cluster = ignite.cluster();
+
+        cluster.active(true);
+
+        ignite.getOrCreateCache("test-cache");
+
+        assertWithPoll(() -> {
+            List<CacheInfo> cacheInfos = interceptor.getListPayload(buildClusterCachesInfoDest(cluster.id()), CacheInfo.class);
+
+            CacheInfo actual = F.find(cacheInfos, null,
+                (P1<CacheInfo>)i -> "test-cache".equals(i.getName()));
+
+            return actual != null &&
+                CU.cacheId("test-cache") == actual.getCacheId() && !actual.isCreatedBySql();
```

Comment:
Let's also assert that they both are not null

Reply:
CU.cacheId("test-cache") from non empty string can't be zero.